exploration/0002-expression-attributes.md

```diff
@@ -38,18 +38,70 @@ At least the following expression attributes should be considered:
 
   - `fallback` — A value to use instead of the default fallback,
     should the expression's primary formatting fail in some way.
+
+    > Example that would format as "Hello world" if `$place` is not defined:
+    >
+    > ```
+    > {Hello {$place @fallback=world}}
```

Comment:
This example is evil? It's basically a string concatenation that requires interior localization?

Reply:
Dropped `@fallback` from the list.